example/lambdas/db-drop-tables/webpack.config.js

```diff
@@ -0,0 +1,45 @@
+const path = require('path');
+const { IgnorePlugin } = require('webpack');
+
+const ignoredPackages = [
+  'mssql',
+  'mssql/lib/base',
+  'mssql/package.json',
+  'mysql',
+  'mysql2',
+  'oracledb',
+  'pg-native',
+  'pg-query-stream',
+  'sqlite3',
+  'tedious'
+];
+
+module.exports = {
+  plugins: [
+    new IgnorePlugin(new RegExp(`^(${ignoredPackages.join('|')})$`))
```

Comment:
Interesting. What created the need to do this? Not that I have an issue with it

Reply:
Webpack was complaining because `knex` was trying to require the npm packages for all of the supported types of databases. I found this solution in a `knex` Github issue.